### PR TITLE
Use Case script for updating Automox devices based on a CSV file source

### PR DESCRIPTION
**Pull request by ax-zyoutz**
# Description
Use Case script for updating Automox devices based on a CSV file source

## Motivation and Context

Fixes # (issue)

### Types of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have updated the documentation accordingly
- [x] I have tested the changes

### CODE REVIEW

File: examples/use-cases/update_devices_by_csv/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Update Automox Devices by CSV
```
Comment (ax-tschmidtke) [inline examples/use-cases/update_devices_by_csv/README.md:1]: Maybe this description could be written a little clearer? This updates the group of a device, correct?

**Review by ax-tschmidtke — DISMISSED** [review]

File: examples/use-cases/update_devices_by_csv/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+A common need is the ability to take a CSV source file to use for grouping a tagging of Automox devices within the 
+Console for filtering, reporting, and scoping needs. This example script shows how to leverage the Automox Console SDK
+to:
```
Comment (ax-zyoutz) [inline examples/use-cases/update_devices_by_csv/README.md:4]:
```suggestion
In order to improve the efficiency and administration of Automox devices, it is possible to leverage a CSV source file and the provided script for the grouping and tagging of Automox devices.  By grouping devices, it makes it that much easier to search for devices, leverage reporting filtering, and scope policies. This script shows how to leverage the Automox Console SDK to:
```

File: examples/use-cases/update_devices_by_csv/README.md (hunk 1 above)
Comment (ax-zyoutz) [reply examples/use-cases/update_devices_by_csv/README.md:1]: Good call, thoughts on proposal below?

File: examples/use-cases/update_devices_by_csv/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+1. Pull Automox Devices
+2. Pull Automox Groups; identifying the default group
+3. Process a CSV by mapping the provided hostname and IP address of each row
+4. Update the matched Automox device by hostname or IP address with the defined field values
```
Comment (ax-zyoutz) [inline examples/use-cases/update_devices_by_csv/README.md:8]:
```suggestion
4. Update the matched Automox devices with the appropriate group and tag values as defined from the CSV
```

File: examples/use-cases/update_devices_by_csv/README.md (hunk 1 above)
Comment (ax-tschmidtke) [reply examples/use-cases/update_devices_by_csv/README.md:1]: These look good!

**Review by ax-tschmidtke — APPROVED** [review]

**Pull request merged**